core/src/main/scala/caliban/interop/tapir/tapir.scala

```diff
@@ -9,7 +9,7 @@ import sttp.tapir.{ Schema, SchemaType }
  */
 private[caliban] trait IsTapirSchema[F[_]]
 private[caliban] object IsTapirSchema {
-  implicit val isCirceEncoder: IsTapirSchema[Schema] = null
+  implicit val isTapirSchema: IsTapirSchema[Schema] = null
```

Comment:
I renamed because it was confusing whether the variable name meant this would only work with the circe encoder. Happy to change it back if it's an issue to rename it

Reply:
Hmm looks like it was a copy paste mistake 😄